airflow/config_templates/default_airflow.cfg

```diff
@@ -349,6 +349,12 @@ celery_app_name = airflow.executors.celery_executor
 # your worker box and the nature of your tasks
 worker_concurrency = 16
 
+# The minimum and maximum concurrency that will be used when starting workers with the
+# "airflow worker" command. Pick these numbers based on resources on
+# worker box and the nature of the task. If autoscale option is available worker_concurrency
+# will be ignored
+#worker_autoscale = 12,16
```

Comment:
Can you add the link to: http://docs.celeryproject.org/en/latest/reference/celery.bin.worker.html#cmdoption-celery-worker-autoscale

Reply:
@Fokko Link has been added. Please check.

Reply:
@phani8996 a space after `#` would be better. i.e `# worker_autoscale = 12,16`